docs/configuration/backends/docker.md

```diff
@@ -169,11 +169,33 @@ Labels can be used on containers to override default behaviour.
 | `traefik.docker.network`                                  | Set the docker network to use for connections to this container. If a container is linked to several networks, be sure to set the proper network name (you can check with `docker inspect <container_id>`) otherwise it will randomly pick one (depending on how docker is returning them). For instance when deploying docker `stack` from compose files, the compose defined networks will be prefixed with the `stack` name. |
 | `traefik.frontend.redirect=https`                         | Enables Redirect to another entryPoint for that frontend (e.g. HTTPS)                                                                                                                                                                                                                                                                                                                                                           |
 
+#### Security Headers
+
+| Label                                                     | Description                                                                                                                                                                                                                                                                                                                                                                                                                     |
+|-----------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `traefik.frontend.headers.allowedHosts=EXPR`              | Provides a list of allowed hosts that requests will be processed. Format: `Host1,Host2`                                                                                                                                                                                                                                                                                                                                                     |
+|`traefik.frontend.headers.customrequestheaders=EXPR `     | Provides the container with custom request headers that will be appended to each request forwarded to the container. Format:  `HEADER:value,HEADER2:value2`                                                                                                                                                                                                            |
```

Comment:
are these meant to be lowercase or camel case?

`traefik.frontend.headers.customrequestheaders` should be `traefik.frontend.headers.customRequestHeaders`

Reply:
Please open an issue instead of commented an already merged PR.

Reply:
@vito-c or open a PR. thanks